### ISSUE TITLE
Support for nvim

### ISSUE THREAD

**Issue by KyleWJohnston** [daehruoydeef/Yin-Yang#34]
I have created bash scripts to change programs between light and dark mode. I'm listing lines from these scripts with the intention of adding support for these programs once I understand how Yin-Yang is structured. If someone else wants to add it first, be my guest. The commands below have only been tested on Fedora Linux.

To change nvim to light mode:
`sed -i 's/set background=.*/set background=light/g' ~/.config/nvim/init.vim`

To change nvim to dark mode:
`sed -i 's/set background=.*/set background=dark/g' ~/.config/nvim/init.vim`

**Comment by daehruoydeef** [comment]
I tried to add support for nvim however this options only tells vim what the background should look like, it does not change the background itself. I try to come up with a better solution.

**Comment by KyleWJohnston** [comment]
Good point. I have [vim-colors-solarized](https://github.com/altercation/vim-colors-solarized) installed, which changes the rest of my colors automatically when I change the background.

**Comment by daehruoydeef** [comment]
Ok i tried around and came up with this:
`sed -i 's/colo.*/colo shine/g' ~/.config/nvim/init.vim`
this would change your theme inside the config to a light theme vise versa it works for a dark theme too. However one would need to restart vim for the changes to take effect.

**Comment by mayapony** [comment]
I am using darkman and vim-lumen to toggle vim background

just like this ⬇️

https://github.com/oskarsh/Yin-Yang/assets/62200137/babadb35-d2e6-4935-9b5e-30e3333a0986

**Comment by kolchurinvv** [comment]
echoing @mayapony  - i was gonna say manipulating the theme of nvim should be done inside nvim IMO since that would give you more control over the themes' settings
